workspaces/default/themes/base/assets/styles/swagger-ui-kong-theme.css

```diff
@@ -1057,6 +1071,34 @@
 .swagger-ui .opblock-body .right-side-wrapper pre span.token.italic{font-style:italic !important}
 .swagger-ui .opblock-body .right-side-wrapper pre span.token.entity{cursor:help !important} */
 
+
+.swagger-ui .opblock-summary-get.opblock-summary .opblock-summary-method {
+  background-color: #1456cb;
+}
+
+.swagger-ui .opblock.opblock-delete .opblock-summary-method {
```

Comment:
Can we use a selector consistent with the others here, like `.swagger-ui .opblock-summary-delete.opblock-summary .opblock-summary-method`?